docs/src/main/sphinx/release/release-453.md

```diff
@@ -0,0 +1,137 @@
+# Release 453 (25 Jul 2024)
+
+## General
+
+* Improve accuracy of the `cosine_distance` function. ({issue}`22761`)
+* Improve performance of inequality joins. ({issue}`22521`)
+* Fix incorrect evaluation of repeated non-deterministic functions. ({issue}`22683`)
+* Fix failure to read Hive tables migrated to Iceberg with Apache Spark. ({issue}`11338`)
+* Fix potential failure for queries involving `GROUP BY`, `UNNEST`, and filters
+  over expressions that may produce an error for certain inputs. ({issue}`22731`)
+* Fix planning failure for queries with a filter on an aggregation. ({issue}`22716`)
+* Fix planning failure for queries involving multiple aggregations and `CASE`
+  expressions. ({issue}`22806`)
+* Fix optimizer timeout for certain queries involving aggregations and `CASE`
+  expressions. ({issue}`22813`)
+
+## Security
+
+* Extend `DROP ROLE` to allow `IF EXISTS`. ({issue}`21985`)
+* Improve performance for column-masking with [](/security/opa-access-control). ({issue}`21359`)
+
+## JDBC driver
+
+* Add support to use a certificate from the operating system keystore. ({issue}`22341`)
+* Add support for setting the default [SQL PATH](/sql/set-path). ({issue}`22703`)
+* Allow Trino host URI specification without port for the default ports 80 for
+  HTTP and 443 for HTTPS. ({issue}`22724`)
+
+## CLI
+
+* Add support to use a certificate from the operating system keystore. ({issue}`22341`)
+* Add support for setting the default [SQL PATH](/sql/set-path). ({issue}`22703`)
+* Allow Trino host URI specification without port for the default ports 80 for
+  HTTP and 443 for HTTPS. ({issue}`22724`)
+
+## BigQuery connector
+
+* Improve performance for querying table definitions. ({issue}`22770`)
+
+## Cassandra connector
+
+* Add support for the `execute` procedure. ({issue}`22556`)
+
+## ClickHouse connector
+
+* Add support for the `execute` procedure. ({issue}`22556`)
+
+## Delta Lake connector
+
+* Add support for reading partitioned tables which the partition column type is
```

Comment:
"Add support for reading partition columns whose type changed via [type widening](...)"